docs/wallets/decrediton/decrediton-setup.md

```diff
@@ -130,3 +130,15 @@ Press **Create Wallet** and your wallet setup is complete. You will be taken bac
 1. Select **Create Wallet**. Wait for the blockchain to be downloaded.
 
 1. The blockchain will now be scanned for any transactions belonging to this wallet.
+
+## Restore Existing Wallet from file
+
+Decred wallet file locations are:
+
+Windows: C:\Users\{Username}\AppData\Local\Decrediton\wallets\mainnet
+
+Mac: ~/Library/Application Support/decrediton/wallets/mainnet/
+
+In order to restore a wallet from file, you can copy and paste the wallet folder to the new computer. 
```

Comment:
Suggest changing the wording to the following:
(I believe the recommended line-length is 80 char)

```
To restore a wallet from a file, it's possible to copy and paste the wallet
folder to the new computer.

However, if you are restoring a wallet which was created on a different
operating system, the config files `dcrwallet.conf` will be incorrect.

To remedy this:

1. Using Decrediton, create a new wallet to populate the correct config files.

1. Close Decrediton.

1. Copy your previous `wallet.db` file and replace it into the new wallet's
mainnet folder: `.../{walletname}/mainnet/wallet.db`

1. Ensure that the new `dcrwallet.conf` settings are correct.

1. Open Decrediton to see your wallet restored.
```